src/syft/lib/python/__init__.py

```diff
@@ -73,6 +76,20 @@ def create_python_ast(client: Optional[AbstractNodeClient] = None) -> Globals:
     ]
 
     methods = [
+        # Slice methods - quite there
+        ("syft.lib.python.Slice.__getitem__", "syft.lib.python.Any"),
```

Comment:
I don't see getitem and iter on the slice Class. Am I missing something?

Reply:
I still get this error. They are present in the slice Class.